.github/workflows/android-common-with-lint.yml

```diff
@@ -51,9 +51,4 @@ jobs:
             signing-key-password: ${{ secrets.ANDROID_SIGNING_KEY_PASSWORD }}
         - name: Generate APK
           shell: bash
-          run: ./gradlew assembleRelease
-        - name: "Publishing to apps.quickbirdstudios.com"
-          uses: QuickBirdEng/actions/qb-publish@main
-          with:
-            upload-key: ${{ secrets.UPLOAD_KEY }}
-            file: app/build/outputs/apk/release/app-release.apk
\ No newline at end of file
+          run: ./gradlew assembleRelease
```

Comment:
Not sure if we should assembleRelease or just assemble

Reply:
The new workflow uses release artifact so why do we need the debug artifact?